# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1288900039 25200
#      Thu Nov 04 12:47:19 2010 -0700
# Node ID 364ef2eb2499e470cd6b48fe4d39ad65522fdd3f
# Parent  452dc55cc55612e2961c012365c755275b970802
Add __fltk_ginput__.m to list of scripts

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-11-04  Rik  <octave@nomad.inbox5.com>
+
+	* plot/module.mk: Add __fltk_ginput__.m to list of scripts
+
 2010-11-04  Rik  <octave@nomad.inbox5.com>
 
 	* Makefile.am: Update check-m-sources target to include generated
 	  function files.
 
 2010-11-04  Rik  <octave@nomad.inbox5.com>
 
 	* general/accumarray.m, general/accumdim.m, general/blkdiag.m,
diff --git a/scripts/plot/module.mk b/scripts/plot/module.mk
--- a/scripts/plot/module.mk
+++ b/scripts/plot/module.mk
@@ -26,16 +26,17 @@ plot_PRIVATE_FCN_FILES = \
   plot/private/__plt__.m \
   plot/private/__pltopt__.m \
   plot/private/__quiver__.m \
   plot/private/__scatter__.m \
   plot/private/__stem__.m \
   plot/private/__tight_eps_bbox__.m
 
 plot_FCN_FILES = \
+  plot/__fltk_ginput__.m \
   plot/__gnuplot_get_var__.m \
   plot/__gnuplot_ginput__.m \
   plot/__gnuplot_has_feature__.m \
   plot/__gnuplot_open_stream__.m \
   plot/__gnuplot_version__.m \
   plot/__go_close_all__.m \
   plot/__go_draw_axes__.m \
   plot/__go_draw_figure__.m \
